projects/demo-integrations/cypress/tests/desktop/kit/input-phone/input-phone.cy.ts

```diff
@@ -0,0 +1,13 @@
+describe(`InputPhone`, () => {
+    it(`filler`, () => {
+        cy.tuiVisit(
+            `components/input-phone/API?tuiTextfieldCustomContent=tuiIconMastercardMono&phoneMaskAfterCountryCode=(%23%23%23)%20%23%23%23-%23%23-%23%23&tuiTextfieldCleaner=true&focusable=true&tuiTextfieldPrefix=&tuiTextfieldPostfix=&tuiTextfieldFiller=57567567&tuiTextfieldSize=l&search=q3e`,
+            {skipExpectUrl: true, skipDecodingUrl: true},
+        );
+
+        cy.get(`#demo-content`)
+            .findByAutomationId(`tui-primitive-textfield__native-input`)
+            .focus()
+            .matchImageSnapshot(`input-phone-filler`);
```

Comment:
Something wrong

![3939265679-0](https://user-images.githubusercontent.com/35179038/213182544-b23e6323-f40f-4edd-8d8b-2b2842075064.png)

Reply:
(not related to this issue)

Reply:
https://github.com/Tinkoff/taiga-ui/issues/3451